src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -75,4 +81,13 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
       case Failure(e) => failedInstantiation(e)
     }
   }
+
+  def receiveResults(results: Try[Map[String, WdlValue]]): Unit = {
```

Comment:
This turned out to be almost a necessary result of the above comment, but I do need to ask you about that one, I'm not quite getting how it's supposed to work